libtransmission/announcer.cc

```diff
@@ -6,11 +6,11 @@
  *
  */
 
-#include <algorithm> // std::partial_sort
-#include <limits.h> /* INT_MAX */
-#include <stdio.h>
-#include <stdlib.h> /* qsort() */
-#include <string.h> /* strcmp(), memcpy(), strncmp() */
+#include <algorithm>
+#include <climits> /* INT_MAX */
+#include <cstdio>
+#include <cstdlib> /* qsort() */
```

Comment:
Strictly speaking, including C++ counterpart of C header only provides access to declarations in `std::` namespace, not to the ones in global namespace that C header would provide. Basically, you can't call `qsort()` (`::qsort()`) after this and have to use `std::qsort()` instead. Also, these changes have nothing to do with MIN/MAX fixes...

Reply:
Well, eventually these header files might get removed and be replaced with actual C++ header files, such as [<limits>](https://en.cppreference.com/w/cpp/header/limits), `std::qsort` with actual `std::sort`, and so forth.

Reply:
Eventually each `*.cc` file will be visited and all std library calls will have `std::` prefixes or rewritten to new C++ semantic. See my other draft pull request it does exactly that. I am not continuing on #1802 till we merge this and the modernization steps #1801, because they base one on another.

Reply:
I'm not saying we shouldn't do this. What I'm trying to say is that this change is not only irrelevant to the PR topic, but is also incomplete. Something to keep in mind for your future PRs, I guess.